spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,15 +30,15 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile2.0]]
-=== MicroProfile 2.0 (2Q2018)
+[[microprofile2.0.1]]
+=== MicroProfile 2.0.1 (3Q2018)
 
-MicroProfile 2.0 is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 2.0.1 [1] is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
 This is a major new release for MicroProfile since the Java EE dependencies are now based on Java EE 8.
 If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/tree/1.x-branch[1.x branch of MicroProfile].
 
-Based on our time-boxed process, the content for MicroProfile 2.0 will be MicroProfile 1.4 plus Java EE 8.
-Thus, the complete list of functional components for MicroProfile 2.0 includes...
+Based on our time-boxed process, the content for MicroProfile 2.0.1 will be MicroProfile 1.4 plus Java EE 8.
```

Comment:
I would say this includes the same content as MicroProfile 2.0. Otherwise, how do we clarify the relationship between MicroProfile 2.0 and 2.0.1

Reply:
Fair point.  I tried to clarify that in the [1] note.  But, based on your previous comments, I'll reword appropriately.